config/application.rb

```diff
@@ -1,6 +1,19 @@
 require File.expand_path('../boot', __FILE__)
 require File.expand_path('../preinitializer', __FILE__)
-require 'rails/all'
+require "rails"
+%w(
+  active_record
+  action_controller
+  action_view
+  action_mailer
+  active_job
+  sprockets
```

Comment:
I've been looking for some programmatic way to disable the loading of Rake tasks from railties but haven't been successful :/ I'll at least try and find some way to fetch these Rails components, sure. This is exactly what Rails does in its loader though, so I'm not sure if there's a list that can be fetched directly or not.

Sent from my iPhone

> On Dec 8, 2015, at 3:28 PM, Jason Frey <notifications@github.com> wrote:
> 
> In config/application.rb:
> 
> > @@ -1,6 +1,19 @@
> >  require File.expand_path('../boot', __FILE__)
> >  require File.expand_path('../preinitializer', __FILE__)
> > -require 'rails/all'
> > +require "rails"
> > +%w(
> > +  active_record
> > +  action_controller
> > +  action_view
> > +  action_mailer
> > +  active_job
> > +  sprockets
> Is there a way we can ask for this list, as opposed to listing it ourselves? If possible would prefer something like Rails.frameworks - ['test_unti'].each do |framework| or something like that.
> 
> —
> Reply to this email directly or view it on GitHub.
>